mvp.feature

```diff
@@ -1,176 +1,274 @@
-        | Possible file for upload  (Filename, File size) |
-	And the Publication is saved and the title is listed in My Publications and marked as Kladd
-    
-        Examples:
-            | Author | Institution | CorrespondingAuthor |
-            | Name Nameson (Not found) | NTNU (Found) | True |
-            | Jan Jansson (Found) | MTNU (Not Found) | False |
-        
-        Examples:
-            | ContributionType | Name | Institution |
-            | Photographer | Jim Jimsson | AVH (Not found) |            
-
-  Scenario: User adds NPI data for a Publication based on a Link
-    Given the the user registers initial metadata for a Publication based on a Link
-    When they click and select a value in the drop-down for NPI subject area
-    Then this is added to the metdata for the Publication
-
-  Scenario: User associates a Publication with a Project they have previously registered a Publication
+    And they click My Publications
+    Then they see the Publication is saved and the title is listed (filename) and marked as Kladd
+
+  @229
+  Scenario: User verifies initial metadata for a Publication based on a Link
+    Given that the user begins registering with a Link
+    And they see that the title metadata for the Link is correct
+    When they click Start
+    And the Publication metadata registration page loads
+    Then they see that the Description tab is populated with metadata values for
+      | Title                            |
+      | Alternative title(s)             |
+      | Abstract                         |
+      | Alternative abstract(s)          |
+      | Description                      |
+      | Publication date                 |
+      | NPI-fagfelt                      |
+      | Keywords                         |
+      | Primary language for publication |
+    And they they see that the Reference tab is populated with metadata values for
+      | Publication type                              |
+      | Link (the link that was provided by the user) |
+      | The unmapped text for Journal                 |
+      | Volume                                        |
+      | Issue                                         |
+      | Page from                                     |
+      | Page to                                       |
+      | Article number                                |
+      | Peer-review                                   |
+    And they see that the Contributors tab Authors section is populated with metadata values for <Author> and <Institution>
+    And they see that the Contributors tab Contributors section is populated with metadata values for <ContributionType>, <Name> and <Institution>
+    And they see that the Files and Licenses tab is pre-populated with metadata values for
+      | Possible mapped value for License, Archiving policy and Unit agreement |
+      | Possible file for upload  (Filename, File size)                        |
```

Comment:
Kan denne featuren splittes opp i en feature for hver tab? synes den er veldig stor